config/api_defaults.yml

```diff
@@ -24,7 +24,7 @@ license: Apache-2.0
 semver:
   go: '0.6.0'
   objc: '0.6.0'
-  java: '0.6.0'
+  java: '0.0.7'
```

Comment:
no I mean 0.0.7. The latest version we have used for the generated grpc Java packages is 0.0.6.